modules/criteoBidAdapter.js

```diff
@@ -268,6 +273,45 @@ export const spec = {
       });
     }
 
+    if (isArray(body.ext?.igbid)) {
+      const seller = body.ext.seller || FLEDGE_SELLER_DOMAIN;
+      const sellerTimeout = body.ext.sellerTimeout || FLEDGE_SELLER_TIMEOUT;
+      const sellerSignals = body.ext.sellerSignals || {};
+      body.ext.igbid.forEach((igbid) => {
+        const perBuyerSignals = {};
+        igbid.igbuyer.forEach(buyerItem => {
+          perBuyerSignals[buyerItem.origin] = buyerItem.buyerdata;
+        });
+        const bidRequest = find(request.bidRequests, b => b.bidId === igbid.impid);
```

Comment:
we don't need to support IE anymore; you shouldnt need this polyfill

Reply:
Thanks, I removed it